projects/angular/src/utils/list-key-navigation/providers/key-navigation-list.config.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2016-2022 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { InjectionToken, ValueProvider } from '@angular/core';
+
+export const ClrListKeyNavigationConfigToken = new InjectionToken('ClrListKeyNavigationConfig');
+
+export const ClrListKeyNavigationConfig = {
+  provide: ClrListKeyNavigationConfigToken,
+  useValue: {
+    keyListItems: 'keyListItems',
+    layout: 'vertical',
+    manageFocus: true,
+    manageTabindex: true,
+    loop: false,
+    dir: null,
+  },
+} as ValueProvider;
```

Comment:
This should be a type annotation instead of a type assertion b/c a type assertion will not generate a compile-time error here if a property is missing/misspelled.

```typescript
export const CLR_LIST_KEY_NAVIGATION_CONFIG_PROVIDER: ValueProvider = { ... };
```